cmd/kms-router.go

```diff
@@ -0,0 +1,81 @@
+// Copyright (c) 2015-2022 MinIO, Inc.
+//
+// This file is part of MinIO Object Storage stack
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU Affero General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU Affero General Public License for more details.
+//
+// You should have received a copy of the GNU Affero General Public License
+// along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+package cmd
+
+import (
+	"net/http"
+
+	"github.com/gorilla/mux"
+	"github.com/klauspost/compress/gzhttp"
+	"github.com/klauspost/compress/gzip"
+	"github.com/minio/minio/internal/logger"
+)
+
+const (
+	kmsPathPrefix       = minioReservedBucketPath + "/kms"
+	kmsAPIVersion       = "v1"
+	kmsAPIVersionPrefix = SlashSeparator + kmsAPIVersion
+)
+
+// KMSAPIHandlers provides HTTP handlers for MinIO KMS API.
+type KMSAPIHandlers struct{}
```

Comment:
Do we need this to be exported?
```suggestion
type kmsAPIHandlers struct{}
```